app/src/main/java/io/mochahub/powermeter/exercises/ExerciseFragment.kt

```diff
@@ -0,0 +1,50 @@
+package io.mochahub.powermeter.exercises
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import androidx.lifecycle.Observer
+import androidx.recyclerview.widget.LinearLayoutManager
+import io.mochahub.powermeter.R
+import kotlinx.android.synthetic.main.fragment_exercise.*
+
+class ExerciseFragment : Fragment() {
+
+    private val viewModel: ExerciseViewModel by viewModels()
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.fragment_exercise, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
```

Comment:
Is this preferred over doing the logic in `onActivityCreated`?

Reply:
Lol idk I think there is no difference 🤷‍♂

Reply:
kk. I'll use oneViewCreated on mine then, to keep it consistent.